modules/administration-guide/partials/con_external-postgresql-setup.adoc

```diff
@@ -0,0 +1,34 @@
+[id="external-postgresql-setup_{context}"]
+= External PostgreSQL setup
+
+By default, {prod-short} Operator creates and manages deployment of the internal PostgreSQL database. However, {prod-short} Operator does not support full life-cycle capabilities such as backups and recovery.
+
+An external PostgreSQL database is recommended for a business-critical setup. The external PostgreSQL can be configured for High Availability (HA) and Point In Time Recovery (PITR).
+
+The external PostgreSQL instance can be configured as on-premises or to use a cloud service such as Amazon Relational Database Service (Amazon RDS). Amazon RDS, for example, enables deployment of production databases in a Multi-Availability Zone configuration for a resilient disaster recovery strategy with daily and on-demand snapshots.
```

Comment:
```suggestion
You can configure the external PostgreSQL instance as on premises or to use a cloud service such as Amazon Relational Database Service (Amazon RDS). Using Amazon RDS, for example, you can deploy production databases in a Multi-Availability Zone configuration for a resilient disaster recovery strategy with daily and on-demand snapshots.
```
- Avoid passive voice. Make the sentence user-centric.
- Per ISG: `Do not use. Use "on-premises" or "on-prem" as an adjective before a noun. Otherwise, use "on premises" or "on prem". If capitalized for a product name, write "On-Premises" or "On Premises". See also "on prem".`